tide_core.module

```diff
@@ -263,6 +264,61 @@ function tide_core_editor_js_settings_alter(array &$settings) {
   }
 }
 
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Move some core fields to Node form Sidebar.
+ */
+function tide_core_form_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $form['#attached']['library'][] = 'tide_core/sticky_node_form_sidebar';
+  $form['#process'][] = '_tide_core_form_node_form_process';
+}
+
+/**
+ * Node form #process callback.
+ *
+ * @param array $element
+ *   Form that is being processed.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   The current state of the form.
+ * @param array $form
+ *   The complete form structure.
+ *
+ * @return array
+ *   The processed form.
+ */
+function _tide_core_form_node_form_process(array $element, FormStateInterface $form_state, array $form = []) {
+  /** @var \Drupal\node\NodeInterface $node */
+  $node = $form_state->getFormObject()->getEntity();
+  $element['tide_core_metadata'] = [
```

Comment:
That is a nice way to do it. I was thinking of doing it through the field_group option, but this way is much easier and cleaner. :+1: